src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Samples.cshtml.cs

```diff
@@ -63,11 +59,17 @@ public async Task<IActionResult> OnGetAsync(string id, string revisionId = null)
             // This try-catch is for the case that the deployment is set up incorrectly for usage samples
             try
             {
-                var sampleFromServer = (await _samplesManager.GetReviewUsageSampleAsync(id)).FirstOrDefault();
-                Samples = sampleFromServer;
-                if (sampleFromServer != null)
+                // Tests the blob response with a dummy file id 
+                string blobTest = await _samplesManager.GetUsageSampleContentAsync("abdc");
```

Comment:
IS this left over from testing?

Reply:
This is here intentionally to catch the case where the blob does not exist but the usage sample Cosmos DB does

If the blob doesn't exist, I only get an error on an attempted fetch OR attempted upload, but since the page loads normally if no usage sample is present, there is potential for an attempted upload to a blob that doesn't exist that causes a regular error, rather than the "Usage Samples are not available in this deployment" message. 

It would be more efficient in terms of performance to move this to the case where no samples are present, though, as that is when the upload may happen before the fetch is done, so I will make that change in the next commit